### ISSUE TITLE
Refractory oral candidiasis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6411]
**Preferred term label:**
Refractory oral candidiasis

**Synonyms**
Refractory oral thrush 

**Definition (free text, please give PubMed ID)**
Refractory accumulation and overgrowth of the fungus Candida albicans on the mucous membranes of the mouth, generally manifested as associated with creamy white lesions on the tongue or inner cheeks, occasionally spreading to the gums, tonsils, palate or oropharynx.
 PMID:  25675017 

**Parent term (use hpo.jax.org/app)**
Oral candidiasis

**Comment by pnrobinson** [comment]
Please use the clinical modifier for "Refractory" (https://hpo.jax.org/app/browse/term/HP:0031375) plus the term for oral candidiasis.
We are evolving the HPO to use modifiers rather than having separate terms for clinical modifiers.
